tasks/58_missed-letters/missed-letters.js

```diff
@@ -0,0 +1,14 @@
+export function findLetter(sentence) {
+  let missedLetters = sentence.split("").reduce((missed, letter, index) => {
+    let [previousCode, currentCode] = [
+      sentence.charCodeAt(index - 1),
+      letter.charCodeAt(0)
+    ];
+
+    return !isNaN(previousCode) && currentCode !== previousCode + 1
```

Comment:
it's more readable if you wrap `previousCode + 1` in brackets. You should not think priority of operators then.
```js
return !isNaN(previousCode) && currentCode !== (previousCode + 1)
```